pages/proposals/language/English.json

```diff
@@ -6,7 +6,7 @@
     "background": {
       "header": "Welcome to the MozFest 2017 Call for Proposals",
       "body": [
-        "MozFest sessions cover a wide range of topics, but share two important qualities: they are <em>interactive</em> and <em>inclusive</em>.",
+        "MozFest sessions cover a wide range of topics, but share two important qualities: they are <em>interactive</em> and <em>inclusive</em>. We welcome workshops and exhibits, and often shy away from sessions that are exclusively lectures and slide presentations.",
```

Comment:
@Pomax thanks for reviewing the content as well as giving suggestions! I'd say let's not review content in this PR as copy changes will have to involve more people for discussion (Kevin, Sarah, Erika, Theo(+localizers), and others). Deadline is approaching so let's aim to have this live as soon as we can first.